ppr-ui/src/composables/mhrRegistration/useMhrCorrections.ts

```diff
@@ -71,6 +74,39 @@ export const useMhrCorrections = () => {
     await goToRoute(RouteNames.SUBMITTING_PARTY)
   }
 
+  /**
+   * Corrects the details of a given home section.
+   * This function modifies the `homeSectionToCorrect` object directly by adjusting its
+   * `action` property based on certain conditions and deep comparisons with baseline data.
+   *
+   * Note: The `homeSectionToCorrect` object is passed by reference.
+   *
+   * @param {HomeSectionIF} homeSectionToCorrect - The home section object to be directly corrected
```

Comment:
This is an important comment, thank you for adding this. Helps future maintainers understand that this is a mutation on the object passed.

Reply:
Yea, I wasn't sure if to pass be _ref_ or _value_ and ended up using the _ref_.